lib/presentation/ui/contributors/contributors_page.dart

```diff
@@ -0,0 +1,113 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class Contributors extends StatefulWidget {
+  @override
+  _ContributorsState createState() => _ContributorsState();
+}
+
+class _ContributorsState extends State<Contributors> {
+  List<Contributor> _contributors;
+
+  @override
+  void initState() {
+    super.initState();
+    _loadContributors();
+  }
+
+  _loadContributors() async {
+    var jsonListContributors = jsonDecode(await DefaultAssetBundle.of(context)
+        .loadString('assets/data/contributors.json'));
+
+    List<Contributor> contributors = List<Contributor>.from(
+        jsonListContributors.map((m) => Contributor.fromJson(m)).toList());
+
+    this.setState(() {
+      _contributors = contributors;
+    });
+  }
+
+  void _openContributorPage(String url) async {
+    if (await canLaunch(url)) {
+      await launch(url);
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Contribuidores"),
+      ),
+      body: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: <Widget>[
+          Padding(
+            padding: const EdgeInsets.all(16.0),
+            child: Text(
+                "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam tincidunt."),
+          ),
+          Expanded(
+            child: ListView.separated(
+              separatorBuilder: (context, index) => Divider(
+                indent: 80.0,
+                thickness: 1.0,
+              ),
+              itemCount: _contributors.length,
```

Comment:
This is giving me the current stacktrace:

```
I/flutter (28903): ══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
I/flutter (28903): The following NoSuchMethodError was thrown building Contributors(dirty, state:
I/flutter (28903): _ContributorsState#139b3):
I/flutter (28903): The getter 'length' was called on null.
I/flutter (28903): Receiver: null
I/flutter (28903): Tried calling: length
I/flutter (28903): 
I/flutter (28903): Widget creation tracking is currently disabled. Enabling it enables improved error messages. It can
I/flutter (28903): be enabled by passing `--track-widget-creation` to `flutter run` or `flutter test`.
I/flutter (28903): 
I/flutter (28903): When the exception was thrown, this was the stack:
I/flutter (28903): #0      Object.noSuchMethod (dart:core-patch/object_patch.dart:51:5)
I/flutter (28903): #1      _ContributorsState.build (package:vost/presentation/ui/contributors/contributors_page.dart:58:40)
I/flutter (28903): #2      StatefulElement.build (package:flutter/src/widgets/framework.dart:4047:27)
I/flutter (28903): #3      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3941:15)
I/flutter (28903): #4      Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #5      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #6      StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4088:11)
I/flutter (28903): #7      ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #8      Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #9      Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #10     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (28903): #11     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #12     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #13     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #14     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #15     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #16     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #17     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #18     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #19     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (28903): #20     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #21     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #22     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (28903): #23     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #24     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #25     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (28903): #26     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #27     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #28     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (28903): #29     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #30     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #31     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #32     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #33     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #34     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4088:11)
I/flutter (28903): #35     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #36     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #37     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #38     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #39     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #40     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #41     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #42     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #43     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #44     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #45     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #46     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #47     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4088:11)
I/flutter (28903): #48     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #49     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #50     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #51     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (28903): #52     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #53     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #54     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #55     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #56     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #57     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #58     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #59     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #60     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (28903): #61     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #62     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #63     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #64     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #65     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #66     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4088:11)
I/flutter (28903): #67     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #68     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #69     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #70     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #71     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #72     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #73     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #74     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #75     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #76     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5127:14)
I/flutter (28903): #77     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #78     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #79     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #80     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #81     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #82     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #83     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #84     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #85     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #86     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #87     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #88     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4088:11)
I/flutter (28903): #89     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #90     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #91     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #92     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #93     Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #94     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:3924:5)
I/flutter (28903): #95     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4088:11)
I/flutter (28903): #96     ComponentElement.mount (package:flutter/src/widgets/framework.dart:3919:5)
I/flutter (28903): #97     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3101:14)
I/flutter (28903): #98     Element.updateChild (package:flutter/src/widgets/framework.dart:2904:12)
I/flutter (28903): #99     RenderObjectElement.updateChildren (package:flutter/src/widgets/framework.dart:4910:32)
I/flutter (28903): #100    MultiChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:5243:17)
I/flutter (28903): #101    Element.updateChild (package:flutter/src/widgets/framework.dart:2893:15)
I/flutter (28903): #102    _TheatreElement.update (package:flutter/src/widgets/overlay.dart:607:16)
I/flutter (28903): #103    Element.updateChild (package:flutter/src/widgets/framework.dart:2893:15)
I/flutter (28903): #104    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #105    Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #106    StatefulElement.update (package:flutter/src/widgets/framework.dart:4120:5)
I/flutter (28903): #107    Element.updateChild (package:flutter/src/widgets/framework.dart:2893:15)
I/flutter (28903): #108    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #109    Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #110    ProxyElement.update (package:flutter/src/widgets/framework.dart:4254:5)
I/flutter (28903): #111    _InheritedNotifierElement.update (package:flutter/src/widgets/inherited_notifier.dart:94:11)
I/flutter (28903): #112    Element.updateChild (package:flutter/src/widgets/framework.dart:2893:15)
I/flutter (28903): #113    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:5134:14)
I/flutter (28903): #114    Element.updateChild (package:flutter/src/widgets/framework.dart:2893:15)
I/flutter (28903): #115    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #116    Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #117    StatefulElement.update (package:flutter/src/widgets/framework.dart:4120:5)
I/flutter (28903): #118    Element.updateChild (package:flutter/src/widgets/framework.dart:2893:15)
I/flutter (28903): #119    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:5134:14)
I/flutter (28903): #120    Element.updateChild (package:flutter/src/widgets/framework.dart:2893:15)
I/flutter (28903): #121    SingleChildRenderObjectElement.update (package:flutter/src/widgets/framework.dart:5134:14)
I/flutter (28903): #122    Element.updateChild (package:flutter/src/widgets/framework.dart:2893:15)
I/flutter (28903): #123    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #124    Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #125    StatelessElement.update (package:flutter/src/widgets/framework.dart:4016:5)
I/flutter (28903): #126    Element.updateChild (package:flutter/src/widgets/framework.dart:2893:15)
I/flutter (28903): #127    ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:3961:16)
I/flutter (28903): #128    Element.rebuild (package:flutter/src/widgets/framework.dart:3738:5)
I/flutter (28903): #129    BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2348:33)
I/flutter (28903): #130    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:760:20)
I/flutter (28903): #131    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:280:5)
I/flutter (28903): #132    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1033:15)
I/flutter (28903): #133    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:975:9)
I/flutter (28903): #134    SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:891:5)
I/flutter (28903): #138    _invoke (dart:ui/hooks.dart:249:10)
I/flutter (28903): #139    _drawFrame (dart:ui/hooks.dart:207:3)
I/flutter (28903): (elided 3 frames from package dart:async)
I/flutter (28903): 
I/flutter (28903): ════════════════════════════════════════════════════════════════════════════════════════════════════

```